src/data.json

```diff
@@ -742,7 +742,7 @@
     "rpccookiefile": {
       "name": "RPC Cookie Directory",
       "description": "Location of the RPC auth cookie",
-      "default": "$BASE/"
+      "default": "$BASE/.cookie"
```

Comment:
Actually upon closer reading of the documentation, I think that the default value should not include $BASE because it is assumed. As such, default value is just ".cookie"

https://github.com/bitcoin/bitcoin/blob/bccb4d29a8080bf1ecda1fc235415a11d903a680/src/rpc/protocol.cpp#L64

"Relative paths will be prefixed by a net-specific datadir location. (default: data dir)" https://github.com/bitcoin/bitcoin/blob/master/src/init.cpp#L505